parm/README.make_flatfile

```diff
@@ -0,0 +1,16 @@
+To make a .txt flat file for UPP, follow these steps within the parm/ directory:
+
+1) Select, then validate, the associated "post_avblflds" file, e.g.,
+
+      xmllint --noout --schema EMC_POST_Avblflds_Schema.xsd fv3lam_post_avblflds.xml
+
+2) Select, then validate, the associated "postcntrl" file, e.g.,
+
+      xmllint --noout --schema EMC_POST_CTRL_Schema.xsd fv3lam_rrfs.xml
+
+3) If file validation succeeds in steps (1) and (2), generate the .txt flat file, e.g.,
+
+      perl PostXMLPreprocessor.pl fv3lam_rrfs.xml fv3lam_post_avblflds.xml postxconfig-NT-fv3lam_rrfs.txt
```

Comment:
explain which ones are input and which one is output should helps.

Reply:
@hu5970 ...thanks!  See if this updated file is clearer.